### ISSUE TITLE
Bindings for Script Node

### ISSUE THREAD

**Issue by hobbyquaker** [hobbyquaker/node-red-contrib-ccu#6]
https://github.com/hobbyquaker/RedMatic/issues/9

**Comment by hobbyquaker** [comment]
@Hypnos3 besteht da noch Bedarf oder kann ich das schließen? Ich denk der Workaround hier HM-RedMatic/RedMatic#9 ist ja zumutbar, oder?

**Comment by Hypnos3** [comment]
Ja, kann geschossen werden. Verwende keine Homematic Scripte mehr.
